client/starwhale/utils/fs.py

```diff
@@ -121,6 +122,39 @@ def blake2b_file(fpath: t.Union[str, Path]) -> str:
     return _hash.hexdigest()
 
 
+def cmp_file_content(
+    base_path: t.Union[str, Path], cmp_path: t.Union[str, Path]
+) -> t.List[str]:
+    base_path = Path(base_path)
+    cmp_path = Path(cmp_path)
+    res = []
+    with base_path.open("r") as base:
+        with cmp_path.open("r") as cmp:
+            diff = difflib.unified_diff(
+                base.readlines(),
+                cmp.readlines(),
+                fromfile=os.path.basename(base_path),
+                tofile=os.path.basename(cmp_path),
+                n=0,
+            )
+            for line in diff:
+
+                def skip(
+                    content: str, overview: bool = True, unchanged: bool = True
```

Comment:
skip function is a inner function, and `overview`, 'unchanged' arguments are not used, so we remove the useless code?

Reply:
removed